apps/new/widget/components/project/Card.jsx

```diff
@@ -126,22 +128,28 @@ const EditButton = ({ item }) => {
 };
 
 const Tags = ({ tags, location }) => {
+  const tagsLength = tags?.length;
+  const firstThreeTags = tagsLength > 3 ? tags.slice(0, 3) : tags;
+  const remainingTags = tagsLength > 3 ? tags.slice(3) : [];
```

Comment:
why are we doing this, we can show multiple tags in one line right? we should three lines (with whatever tags it can contain), and then for remaining using either +{length}, or show "See More" or something like that.